src/main/java/org/medicmobile/webapp/mobile/EmbeddedBrowserActivity.java

```diff
@@ -51,8 +49,7 @@
 	static final int GRAB_PHOTO = (0 << 3) | NON_SIMPRINTS_FLAGS;
 	static final int GRAB_MRDT_PHOTO = (1 << 3) | NON_SIMPRINTS_FLAGS;
 
-	private static final long FIVE_MINS = 5 * 60 * 1000;
-	private static final float ANY_DISTANCE = 0f;
+	private final static int ACCESS_FINE_LOCATION_PERMISSION_REQUEST = 1;
```

Comment:
This is supposed to be application specific, and 1 will probably be used by other apps, so maybe make it a randomish number?

Reply:
sure thing! I'll change it to something else.

Reply:
Done!